langchain/llm.go

```diff
@@ -44,14 +45,20 @@ func cleanResponse(resp string) string {
 func (c Client) SingleMessageResponse(ctx context.Context, msg database.TwitchMessage) (string, error) {
 	prompt, err := c.callLLM(ctx, []string{fmt.Sprintf("%s: %s", msg.Username, msg.Text)})
 	if err != nil {
+		metrics.FailedLLMGen.Add(1)
 		return "", err
 	}
+	if prompt == "" {
+		metrics.EmptyLLMResponse.Add(1)
+		return fmt.Sprintf("sorry, I cannot respont to @%s. Please try again", msg.Username), nil
+	}
```

Comment:
maybe add a comment that we are trying to get a better prompt from the user and that is why we are using the `@` to call them out.